### PR TITLE
docs(readme): add adapter explanation and usage

### PR DESCRIPTION
**Pull request by lalli-flores**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,36 +35,68 @@ npm install --save @webex/components
...
+### Adapters
+Webex Components are self-updating, meaning, they know how to fetch the data they need.
+Data is passed to components via adapter classes (see [adapters](./src/adapters)).
+Adapters are an uniform interface for the Webex Components to consume.
+They also know how to map the data from their data source to the data the components need.
 
+To use a Webex Component, start by creating a Webex Adapter:
 ```js
-import {WebexAvatar} from '@webex/components';
+import '@webex/components/dist/webexComponents.css';
```
Comment (akoushke) [inline README.md]: I dont think this is necessary. you have it imported in your components file.

Comment (lalli-flores) [reply README.md]: What components file?
The idea is that the sections build on each other: style, style + adapter, style + adapter + component

Comment (akoushke) [reply README.md]: this looks like you are making this file an entry file, which does not have to be. it looks a bit confusing to me. You are introducing the adapter but adding extra code about style import declaration which is not relevant to your description.

Comment (akoushke) [reply README.md]: we are addressing the styling import declaration in the earlier step with two ways. and we should leave it at that.

Comment (akoushke) [reply README.md]: this means that you are basically assuming that all these declaration will occur in `index.js` which potentially will not be the case.

**Pull request merged**